library/src/main/java/com/timehop/stickyheadersrecyclerview/util/OrientationProvider.java

```diff
@@ -7,6 +7,7 @@
  */
 public interface OrientationProvider {
 
-  public int getOrientation(RecyclerView recyclerView);
+    public int getOrientation(RecyclerView recyclerView);
 
+    public boolean getReverseLayout(RecyclerView recyclerView);
```

Comment:
Can we call this `isReverseLayout()`? I realize it's not consistent with `LinearLayoutManager` but I feel like that is their problem :)